_posts/2022-08-04-decade-innovation.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: post
+title:  "A decade of query engine innovation"
+author: "Manfred Moser, Martin Traverso, Dain Sundstrom, David Phillips"
+excerpt_separator: <!--more-->
+---
+
+It's amazing how far we have come! Our massively-parallel processing SQL query
+engine, Trino, has really grown up. We move beyond just querying object stores
+using Hive, beyond just one company using the project, beyond usage in Silicon
+Valley, beyond simple SQL `SELECT` statements, and definitely also beyond our
+expectations. Let’s have a look at some of the great technical and architectural
+changes the project underwent, and how we all benefit from the [commitment to
+quality, openness and collaboration]({% post_url
+2022-08-02-leaving-facebook-meta-best-for-trino %}).
+
+<!--more-->
+
+## Runtime and deployment
+
+Starting with how you even run Trino and install it, numerous changes came about
+in the last decade. We moved from Java 7 to Java 8, then to Java 11, and [only
+recently to the latest supported Java LTS release - Java 17]({% post_url
+2022-07-14-trino-updates-to-java-17 %}). Each time we
+benefited from the innovations in the runtime performance as well as the
+improved Java language features. With **Java 17**, we are just about to start a lot
+of these improvements.
+
+When it comes to actually [running and deploying
+Trino]({{site.url}}/episodes/35.html), the **tarball** is still a good choice
+for simple installation and as a base for other packages. Over time we added
+**RPM** archive support, which is being replaced more and more by Docker
+**containers**. The container images also enable modern deployment on Kubernetes
+with [our Helm chart](https://github.com/trinodb/charts).
+
+And let us add one last note about deployments. It’s been amazing to see how the
```

Comment:
Not sure what point this is trying to make:

> It’s been amazing to see how the machine sizes in terms of CPU and memory keep growing. Some truly large machines are now running as clusters.

If we're trying to say that Trino supports larger machines, then we should say so more directly. However, we haven't done anything specific for this purpose, so I might just remove it.

Reply:
Will reword